src/ngAnimate/animateCss.js

```diff
@@ -223,12 +223,13 @@ var ELAPSED_TIME_MAX_DECIMAL_PLACES = 3;
 var CLOSING_TIME_BUFFER = 1.5;
 
 var DETECT_CSS_PROPERTIES = {
-  transitionDuration:      TRANSITION_DURATION_PROP,
-  transitionDelay:         TRANSITION_DELAY_PROP,
-  transitionProperty:      TRANSITION_PROP + PROPERTY_KEY,
-  animationDuration:       ANIMATION_DURATION_PROP,
-  animationDelay:          ANIMATION_DELAY_PROP,
-  animationIterationCount: ANIMATION_PROP + ANIMATION_ITERATION_COUNT_KEY
+  transitionDuration:       TRANSITION_DURATION_PROP,
+  transitionDelay:          TRANSITION_DELAY_PROP,
+  transitionProperty:       TRANSITION_PROP + PROPERTY_KEY,
+  transitionTimingFunction: TRANSITION_PROP + TIMING_KEY,
```

Comment:
These 2 lines shoould use the variables defined in `shared.js`: `TRANSITION_PROPERTY_PROP` and `TRANSITION_TIMING_PROP`